### PR TITLE
3 more Larky samples

### PR DESCRIPTION
**Pull request by 420neshot**
Added:
- Reading a header
- Working with Path
- Working with Query String

### CODE REVIEW

File: integrations/larky/input/query_string/test-query-string.star [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+vault = {
+    "tok_sandbox_a4JULYBMphkeL2iV7ipwdL": "4111111111111111"
+}
+
+def process(input, ctx):
+    path = input.path
+    query = input.query_string
+    body = json.loads(str(input.body))
+
+    lst = query.split('=')
```
Comment (mottersheadt) [inline integrations/larky/input/query_string/test-query-string.star:23]: Do we have a way of getting a key/value representation of the query string? This example is assuming that the query string keys are always in a specific order.

**Review by mottersheadt — CHANGES_REQUESTED** [review]

File: integrations/larky/input/query_string/test-query-string.star (hunk 1 above)
Comment (420neshot) [reply integrations/larky/input/query_string/test-query-string.star]: came up with https://stackoverflow.com/questions/21584545/url-query-parameters-to-dict-python

**Review by mottersheadt — APPROVED** [review]

File: integrations/larky/input/query_string/inbound-query-string.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                  load("@stdlib//urllib/parse", parse="parse")
+
+                  def process(input, ctx):
+                      query = input.query_string
+                      
+                      body = dict(parse.parse_qsl(query))
+                      
+                      body['account'] = vault.reveal(body['account']) # aka Reveal
+
+                      input.body = builtins.bytes(json.dumps(body))
```
Comment (mottersheadt) [inline integrations/larky/input/query_string/inbound-query-string.yml:36]: This is taking the query string and assigning it to the body whichnwill fail for GET requests. Let's have an example of overwriting the query string instead.

**Review by mottersheadt — CHANGES_REQUESTED** [review]

File: integrations/larky/input/query_string/inbound-query-string.yml (hunk 2 above)
Comment (420neshot) [reply integrations/larky/input/query_string/inbound-query-string.yml:36]: Doesn't look like we have here an opposite method for parse_qsl()
https://github.com/verygoodsecurity/starlarky/blob/master/larky/src/main/resources/stdlib/urllib/parse.star

so came up with FOR loop

**Review by mottersheadt — APPROVED** [review]

**Pull request merged**